inlang/source-code/importers/icu2-message-format/src/plugin.ts

```diff
@@ -0,0 +1,149 @@
+import * as MF from "messageformat"
+import { displayName, description } from "../marketplace-manifest.json"
+import { PluginSettings } from "./settings.js"
+import { AST } from "@inlang/sdk"
+import { Importer } from "../../../versioned-interfaces/importer/dist/interface.js"
+
+export const pluginId = "importer.inlang.icu2MessageFormat"
+
+export const plugin: Importer<{
+	[pluginId]: PluginSettings
+}> = {
+	id: pluginId,
+	displayName,
+	description,
+	settingsSchema: PluginSettings,
+	importMessages: async ({ settings, nodeishFs }) => {
```

Comment:
@jldec this a new function we use to keep the signature of loadMessages (soon to be deprecated) untouched

Reply:
ok - I wonder if there isn't a way to feature flag the api?